Jellyfin.Server/CoreAppHost.cs

```diff
@@ -74,7 +74,8 @@ protected override void RegisterServices()
             }
 
             ServiceCollection.AddDbContextPool<JellyfinDb>(
-                 options => options.UseSqlite($"Filename={Path.Combine(ApplicationPaths.DataPath, "jellyfin.db")}"));
+                 options => options.UseLazyLoadingProxies().
```

Comment:
Why?

Reply:
@cvium I've been stuck for hours trying to check why EFCore wouldn't load the ``Image`` entity related to an user, thinking something was borked in my migration procedures. Rest of the code worked fine (i.e I could login and see my profile image) but somehow I couldn't access ``user.ProfileImage`` (it was null) in a ``Jellyfin.Server`` migration I did for testing that everything was accessible.

Researching about this, I found that EFCore doesn't load related entities by default and that you must explicitly include them (https://docs.microsoft.com/es-es/ef/core/querying/related-data/eager), otherwise they're ``null`` if they're nullable in the code.

Adding ``virtual`` to the properties allows EFCore to override the getter and do the query for you without further work (which I think was the initial design goal of the schema as everything was ``virtual``). However, this needs of LazyLoading (https://docs.microsoft.com/es-es/ef/core/querying/related-data/lazy) and ``Microsoft.EntityFrameworkCore.Proxies`` dependency to work.

So basically I added this so everything is easily accessible out of the box and nobody else spends more time debugging this the same way I did just because I wasn't properly aware of this EFCore feature. I don't mind not reverting the commit as I know now how to do proper queries with ``nullable`` stuff, but I think it's nice to have.

Here you have all the ways of loading data using EFCore: https://docs.microsoft.com/es-es/ef/core/querying/related-data/

Reply:
I don't like lazy loading tbh. It creates separate queries for every relationship when accessed and it doesn't work if you disconnect the entity from the db context. If some unfortunate soul decides to map all properties to a Dto, the lazy loading would bite them in the ass.

Reply:
@cvium Fine, reverted! 👍